extensions/amp-auto-ads/0.1/placement.js

```diff
@@ -187,21 +189,16 @@ export class Placement {
   }
 
   /**
-   * @param {string} type
-   * @param {!Array<!DataAttributeDef>} dataAttributes
+   * @param {!Object<string, string>} baseAttributes
    * @return {!Element}
    * @private
    */
-  createAdElement_(type, dataAttributes) {
-    const attributes = {
-      type,
+  createAdElement_(baseAttributes) {
+    const attributes = Object.assign({
       'layout': 'responsive',
       'width': '0',
       'height': '0',
-    };
-    for (let i = 0; i < dataAttributes.length; ++i) {
-      attributes['data-' + dataAttributes[i].name] = dataAttributes[i].value;
-    }
+    }, baseAttributes, this.attributes_);
```

Comment:
**DANGER**: this is an XSS vulnerability. I can use this to add `onclick`, etc, and execute JS.